target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -223,16 +221,12 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "1:wan" "2:lan:3" "3:lan:2"
 		;;
-	engenius,enh202-v1)
-		ucidef_set_interface_wan "eth0"
-		ucidef_add_switch "switch0" \
-			"0@eth1" "4:lan:1"
-		;;
 	engenius,ens202ext-v1)
-		ucidef_set_interface_wan "eth1"
+		ucidef_set_interface_lan "eth1"
 		ucidef_add_switch "switch0" \
 			"0@eth0" "1:lan"
 		;;
+	engenius,enstationac-v1|\
 	engenius,ews511ap)
 		ucidef_set_interface_lan "eth0 eth1" "dhcp"
 		;;
```

Comment:
a side note: is it possible that someone would have a very hard time with SSH or failsafe mode without a router since DHCP is the default protocol?

Should we consider not having DHCP as default for any board?

Reply:
I just realized this is a bad question, but I still think it is pointless to have the "dhcp", and they should be merged with the other cases